guides/common/modules/proc_configuring-the-open-files-limit-for-apache-httpd.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Configuring_the_Open_Files_Limit_for_Apache_HTTPD_{context}"]
+= Configuring the Open Files Limit for Apache HTTPD
+
+With the tuning in place, Apache httpd can easily open a lot of file descriptors on the server which may exceed the default limit of most of the linux systems in place.
+To avoid any kind of issues that may arise as a result of exceeding max open files limit on the system, please create the following file and directory and set the contents of the file as specified in the below given example:
+
+.Procedure
+. Set the maximum open files limit in `/etc/systemd/system/httpd.service.d/limits.conf`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[Service]
+LimitNOFILE=640000
+----
+. Restart the {Project} services:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl daemon-reload
+# {foreman-maintain} service restart
```

Comment:
I hate this if I'm honest. I really don't know why we don't just use
```suggestion
# systemtl restart httpd
```

There is 0 reason to restart all services and only takes a long time.

I think there is also an option like `--only` to foreman-maintain but really, we're already deep into systemd by editing an override file (it doesn't mention you need to create `/etc/systemd/system/httpd.service.d` for example). It also calls daemon-reload.

Reply:
Applied.